catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -1171,7 +1170,7 @@ private static boolean processEntry(String entryName, List<String> requiredAttrs
                     .filter(requiredAttr -> !parsedAttrs.contains(requiredAttr))
                     .forEach(missingAttr -> {
                         dataIsValid.set(false);
-                        LOGGER.warn("Node: {} is missing attribute: {}", entryName, missingAttr);
+                        LOGGER.trace("Node: {} is missing attribute: {}", entryName, missingAttr);
```

Comment:
This is still pretty noisy and we're setting the thread local for every missing attribute which is silly. How about something like:
```java
List<String> missingAttrs = requiredAttrs.stream()
        .filter(requiredAttr -> !parsedAttrs.contains(requiredAttr))
        .collect(Collectors.toList());
if (!missingAttrs.isEmpty()) {
    dataIsValid.set(false);
    LOGGER.debug("Node: {} is missing attributes: {}", entryName, missingAttrs);
}
```